robot_modeling/DQ_SerialManipulator.m

```diff
@@ -289,48 +194,18 @@ function set_effector(obj,effector)
             % This function does not take into account any base or
             % end-effector displacements.
             
-            if nargin == 4
-                n = ith;
-                x_effector = obj.raw_fkm(q,ith);
-                J = obj.raw_pose_jacobian(q,ith);
-                vec_x_effector_dot = J*q_dot(1:ith);
+            if nargin == 4 && ith < obj.n_links
+                % If the Jacobian is not related to the mapping between the
+                % end-effector velocities and the joint velocities, it takes
+                % into account only the base displacement
```

Comment:
(...) into account only the **constant** base displacement

Reply:
- [ ] Implemented.